projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -47,6 +49,8 @@ export default function PublicizeForm( {
 		! isRePublicizeFeatureEnabled && connections.every( connection => ! connection.toggleable );
 	const Wrapper = isPublicizeDisabledBySitePlan ? Disabled : Fragment;
 
+	const brokenConnections = connections.filter( connection => ! connection.is_healthy );
```

Comment:
I think if we remove `is_healthy` from the post field data, then we might have to default it to `true` or do a strict check, like `false === connection.is_healthy` to make sure that it doesn't look like all the connections are broken before we get the response from the test connections API call.